certora/spec/timelockAuthorizerSasha.spec

```diff
@@ -180,10 +180,10 @@ rule onlyExecuteAndCancelCanChangeTheirFlags(env e, method f) {
 
     assert (isExecuted1Before != isExecuted1After 
                 || isExecuted2Before != isExecuted2After)
-            => f.selector == execute(uint256).selector;
+            <=> f.selector == execute(uint256).selector;
     assert (isCancelled1Before != isCancelled1After 
                 || isCancelled2Before != isCancelled2After)
-            => f.selector == cancel(uint256).selector;
+         /*can we make it stronger? */  <=> f.selector == cancel(uint256).selector;
```

Comment:
Can only in case if we create a helperFunction to call execute and cancel on checked indexes. Otherwise, there is a violation because the function call is parametric and it cancels/executes another index. Or we can use quantifier "exists", but it doesn't work properly I guess